QualityControl/test/public/pages/layout-show.test.js

```diff
@@ -200,7 +200,16 @@ export const layoutShowTests = async (url, page, timeout = 5000, testParent) =>
     'should have number input field for allowing users to change auto-tab value',
     { timeout },
     async () => {
-      await page.waitForSelector('#inputDescription', { timeout: 5000 });
+      const inputSelector = '#inputChangeTabTimer';
```

Comment:
Given that in this test you also add a check for value to be more than 10, I think the test should be split in 2. The current description of the test does not cover both